### PR TITLE
Added refactoring and code clean up for java files

### PR DESCRIPTION
**Pull request by bipin-k**
[x] Added refactoring and code clean up for java files

### CODE REVIEW

File: center/src/test/java/com/microsoft/hydralab/center/service/DeviceGroupServiceTest.java [hunk 1]
```diff
@@ -27,7 +27,7 @@ void testDeviceGroupManagement() {
         SysTeam defaultTeam = sysTeamService.getOrCreateDefaultTeam(Const.DefaultTeam.DEFAULT_TEAM_NAME);
 
         deviceGroupService.createGroup(defaultTeam.getTeamId(), defaultTeam.getTeamName(), groupName, null);
-        DeviceGroup group = deviceGroupService.getGroupByName(Const.DeviceGroup.groupPre + groupName);
+        DeviceGroup group = deviceGroupService.getGroupByName(Const.DeviceGroup.GROUP_PRE + groupName);
```
Comment (hydraxman) [inline center/src/test/java/com/microsoft/hydralab/center/service/DeviceGroupServiceTest.java:30]: Thanks for the help. Here maybe we can use GROUP_NAME_PREFIX for clarity.

**Review by hydraxman — COMMENTED** [review]
Only one comment on the field name. Thanks!

**Comment by bipin-k** [comment]
@microsoft-github-policy-service agree [company="mpl"]

**Comment by bipin-k** [comment]
@microsoft-github-policy-service agree

**Comment by bipin-k** [comment]
@microsoft-github-policy-service agree company="MPL"

**Review by hydraxman — APPROVED** [review]

**Review by zhou9584 — APPROVED** [review]

**Pull request merged**